uaclient/jobs/update_contract_info.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from uaclient import contract, event_logger, messages, util
+from uaclient.config import UAConfig
+from uaclient.files import notices
+from uaclient.files.notices import Notice
+
+LOG = logging.getLogger(__name__)
+event = event_logger.get_event_logger()
```

Comment:
I see the `event.warning` line got copy pasted but it is only needed in cli commands. Because this is executing as part of the timer script, we don't need it.
```suggestion

```

Reply:
Ups, removed.